node/node.go

```diff
@@ -905,7 +905,9 @@ func (app *App) initServices(ctx context.Context, poetClients []activation.PoetP
 			peersync.WithConfig(app.Config.TIME.Peersync),
 		)
 	}
-
+	if err := app.host.Start(); err != nil {
+		return err
+	}
```

Comment:
can be simplified to

```suggestion
	return app.host.Start()
```

Reply:
i intentionally made it so that next person can write their own thing without changing previous code